### ISSUE TITLE
Compiling on Raspberry Pi

### ISSUE THREAD

**Issue by webspacecreations** [AppleCommander/AppleCommander#45]
The current code does not compile on Pi/Arm. It can be made to compile by taking the following actions:

On Pi (Raspbian), issue command:
sudo apt-get install libswt-gtk-3-java

Make the following edits to build.gradle in repo:
Line 111:
- compile "org.eclipse.swt:org.eclipse.swt.gtk.linux.x86_64:$swtVersion"
+ include '/usr/share/java/swt-gtk-3.8.jar'

Line 113:
- appendix 'linux64-gtk'
+ appendix 'linuxarm-gtk'

Note: This approach results in a package that will compile and run, but GUI does not actually list file contents when opening an Apple II disk image. Also, a linux-arm clause would obviously need to be added to the build.gradle file, so that the linux scenario would continue to build for x86. It may also be possible to issue a compile option instead of the include (so that apt-get install isn't necessary), but I wasn't able to track down a URL that would be referenced for the Arm build of SWT-Gtk.

**Comment by a2geek** [comment]
Hey, thanks for the ticket! I've been able to experiment enough that I think I've got something that works. I tinkered a bit with my Pi and it looks good.

Ironically, when I use SWT 4.6.0 (available via `apt`) I can't get it to work. The 3 other distributions are on SWT 4.6.1. But, I tried the version you had and 3.8.2 seems to work fine!

I'll attach a JAR file to this ticket until I figure out where to place the SWT jar file (maybe just in the AppleCommander project itself).
